Cubical/Categories/Constructions/Slice/Functor.agda

```diff
@@ -0,0 +1,193 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Constructions.Slice.Functor where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Isomorphism
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Category.Properties
+
+open import Cubical.Categories.Constructions.Slice.Base
+
+open import Cubical.Categories.Limits.Pullback
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Adjoint
+
+open import Cubical.Tactics.FunctorSolver.Reflection
+
+
+open Category hiding (_∘_)
+open Functor
+open NatTrans
+
+private
+  variable
+    ℓ ℓ' : Level
+    C D : Category ℓ ℓ'
+    c d : C .ob
+
+infix 39 _F/_
+infix 40 ∑_
+
+_F/_ : ∀ (F : Functor C D) c → Functor (SliceCat C c) (SliceCat D (F ⟅ c ⟆))
+F-ob (F F/ c) = sliceob ∘ F ⟪_⟫ ∘ S-arr
+F-hom (F F/ c) h = slicehom _
+  $ sym ( F-seq F _ _) ∙ cong (F ⟪_⟫) (S-comm h)
+F-id (F F/ c) = SliceHom-≡-intro' _ _  $ F-id F
+F-seq (F F/ c) _ _ = SliceHom-≡-intro' _ _  $ F-seq F _ _
+
+
+∑_ : ∀ {c d} f → Functor  (SliceCat C c) (SliceCat C d)
+F-ob (∑_ {C = C} f) (sliceob x) = sliceob (_⋆_ C x f)
+F-hom (∑_ {C = C} f) (slicehom h p) = slicehom _ $
+  sym (C .⋆Assoc _ _ _) ∙ cong (comp' C f) p
+F-id (∑ f) = SliceHom-≡-intro' _ _ refl
+F-seq (∑ f) _ _ = SliceHom-≡-intro' _ _ refl
+
+module _ (Pbs : Pullbacks C) where
+
+ open Category C using () renaming (_⋆_ to _⋆ᶜ_)
```

Comment:
I think the more common style in the library is to use `⋆⟨ C ⟩`, as well as above on line 44.

Reply:
I just put `_⋆⟨_⟩_` where possible, but in other places I am using sections, and unfortunetly I can do  `_ ⋆ᶜ_` but not `_ ⋆⟨ C ⟩_` (secound version does not parse)